LEAF_Nexus/scripts/convertFileNames.php

```diff
@@ -1,106 +0,0 @@
-<?php
```

Comment:
Is this file no longer needed?

Reply:
You can double check but when I search for the file name it does not come up. So best as I can tell it is not used anywhere.